lib/klaviyo/apis/public.rb

```diff
@@ -27,7 +27,7 @@ def self.identify(kwargs = {})
         :properties => properties
       }
 
-      public_request(HTTP_GET, 'identify', **params)
+      public_request(HTTP_POST, 'identify', **params)
```

Comment:
Would it make sense to allow both HTTP verbs for backwards compatibility? Could add an argument to the track/identify functions or perhaps a flag on the Klaviyo singleton object.

Reply:
From the user POV nothing is different, they still call track() or identify() - this update just changes the request down the line.  Is there a reason a user would _need_ to make a get request vs a post request?

Reply:
I'm in favor of not worrying about backward compatibility to support get requests.
The end user is not seeing whats under the hood, moreover this would involve the user to choose what request to make which may lead to confusion on what's happening

Reply:
Just thought I would mention it because the python and node.js SDK PRs for this update are keeping GET as the default HTTP verb and allowing the user to specify POST via a kwarg:
https://github.com/klaviyo/python-klaviyo/pull/46
https://github.com/klaviyo/node-klaviyo/pull/25

Reply:
Hmm, I guess we could talk more on Slack

Reply:
Yea, we want to maintain backwards compatibility and continue to leave GET as the primary codepath, and optionally allow POST.